AutomaticQC/imosEchoIntensityVelocitySetQC.m

```diff
@@ -113,29 +117,66 @@
 % same flags are given to any variable
 flags = ones(sizeCur, 'int8')*rawFlag;
 
+isUpwardLooking = true;
+if all(Bins <= 0), isUpwardLooking = false; end
```

Comment:
Just checking if sample_data.dimensions{idHeight}.data only contains in water data? Just comparing parsers (say line 315 of workhorseParse.m)
```
adcpOrientation = mode(adcpOrientations); % hopefully the most frequent value reflects the orientation when deployed
```

And following on from that, if the parsers have already calculated orientation once (which it looks like aquadoppProfiler, awac, contental, signature, workhorse do) then can't this be saved in the sample_data.meta area for reuse.

Reply:
yes, this should be within the meta struct. I would avoid creating more reviews here @sspagnol because I can still rebase this a bit and the comments be lost.

Reply:
to make sure, just create further comments at the top-level PR page!